helix-core/src/main/java/org/apache/helix/model/Message.java

```diff
@@ -522,12 +538,25 @@ public long getExecuteStartTimeStamp() {
 
   /**
    * Get the time that this message was created
-   * @return UNIX timestamp
+   * @return UNIX epoch timestamp
```

Comment:
This is not epoch time. epoch time is not at milisecond but second level.

Reply:
Fixed